packages/admin-ui/src/pages/packages/components/Network/PortsByService.tsx

```diff
@@ -120,59 +120,101 @@ export function PortsByService({
     return conflictingPorts;
   }
 
-  function getPortsOverTheMax(): PortMapping[] {
+  // Function to get any container ports in the ephemeral range, excluding port 51820
+  function getContainerPortsInEphemeralRange(): PortMapping[] {
     return ports.filter(
-      ({ host, container, deletable }) =>
-        (deletable &&
-          (container > maxPortNumber || (host && host > maxPortNumber))) ||
-        (host && host > maxEphemeralPortNumber)
+      ({ container, deletable }) =>
+        deletable && container > maxRegisteredPortNumber && container <= maxPortNumber && container !== 51820
```

Comment:
I would create a const `wireguardPort` where it is briefly explained, instead of hardcoding 51820 every time

Also, you could even create a function callled `isWireguardPort` which would be more self explanatory each time you check if port is 51820